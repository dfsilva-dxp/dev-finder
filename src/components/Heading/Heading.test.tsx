import { render, screen } from "@testing-library/react";

import Heading from ".";

describe("Heading", () => {
  it("should render currectly", () => {
    const { container } = render(<Heading />);

    expect(
      screen.getByRole("heading", { name: /heading/i })
    ).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
