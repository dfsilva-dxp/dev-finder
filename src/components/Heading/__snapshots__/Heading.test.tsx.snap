// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render currectly 1`] = `
<div>
  <h1>
    Heading
  </h1>
</div>
`;
